test/JDBC/expected/test_db_collation-vu-verify.out

```diff
@@ -0,0 +1,1800 @@
+nt#!#bbf_unicode_cp1_ci_as
+nv#!#bbf_unicode_cp1_ci_ai
+nv#!#bbf_unicode_cp1_ci_ai
+nv#!#bbf_unicode_cp1_ci_as
+nv#!#bbf_unicode_cp1_ci_as
+sn#!#bbf_unicode_cp1_ci_ai
+sn#!#bbf_unicode_cp1_ci_ai
+sn#!#bbf_unicode_cp1_ci_as
+sn#!#bbf_unicode_cp1_ci_as
+sqlv#!#bbf_unicode_cp1_ci_ai
+sqlv#!#bbf_unicode_cp1_ci_ai
+sqlv#!#bbf_unicode_cp1_ci_as
+sqlv#!#bbf_unicode_cp1_ci_as
+t#!#bbf_unicode_cp1_ci_ai
+t#!#bbf_unicode_cp1_ci_ai
+t#!#bbf_unicode_cp1_ci_as
+t#!#bbf_unicode_cp1_ci_as
+v#!#bbf_unicode_cp1_ci_ai
+v#!#bbf_unicode_cp1_ci_ai
+v#!#bbf_unicode_cp1_ci_as
+v#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+select column_name, collation_name FROM information_schema_tsql.COLUMNS WHERE TABLE_NAME = 'test_db_collation_vu_prepare_db142_t1' AND
+COLUMN_NAME IN ('c', 'nc', 'v', 'nv', 't', 'nt', 'sqlv', 'sn') ORDER BY COLLATION_NAME + COALESCE(collation_name, '');
+GO
+~~START~~
+nvarchar#!#nvarchar
+c#!#bbf_unicode_cp1_ci_as
+nc#!#bbf_unicode_cp1_ci_as
+v#!#bbf_unicode_cp1_ci_as
+nv#!#bbf_unicode_cp1_ci_as
+t#!#bbf_unicode_cp1_ci_as
+nt#!#bbf_unicode_cp1_ci_as
+sqlv#!#bbf_unicode_cp1_ci_as
+sn#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+select column_name, collation_name FROM information_schema_tsql.COLUMNS WHERE TABLE_NAME = 'test_db_collation_vu_prepare_db142_t2' AND
+COLUMN_NAME IN ('c', 'nc', 'v', 'nv', 't', 'nt', 'sqlv', 'sn') ORDER BY COLLATION_NAME + COALESCE(collation_name, '');
+GO
+~~START~~
+nvarchar#!#nvarchar
+c#!#bbf_unicode_cp1_ci_ai
+nc#!#bbf_unicode_cp1_ci_ai
+v#!#bbf_unicode_cp1_ci_ai
+nv#!#bbf_unicode_cp1_ci_ai
+t#!#bbf_unicode_cp1_ci_ai
+nt#!#bbf_unicode_cp1_ci_ai
+sqlv#!#bbf_unicode_cp1_ci_ai
+sn#!#bbf_unicode_cp1_ci_ai
+~~END~~
+
+
+SELECT name, collation_name FROM SYS.TYPES WHERE collation_name IS NOT NULL ORDER BY name + COALESCE(collation_name, '');
+GO
+~~START~~
+varchar#!#varchar
+char#!#bbf_unicode_cp1_ci_as
+ct#!#bbf_unicode_cp1_ci_as
+nchar#!#bbf_unicode_cp1_ci_as
+ntext#!#bbf_unicode_cp1_ci_as
+nvarchar#!#bbf_unicode_cp1_ci_as
+sql_variant#!#bbf_unicode_cp1_ci_as
+sysname#!#bbf_unicode_cp1_ci_as
+text#!#bbf_unicode_cp1_ci_as
+varchar#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+CREATE TYPE ct FROM NVARCHAR(11);
+GO
+~~ERROR (Code: 219)~~
+
+~~ERROR (Message: type "ct" already exists)~~
+
+
+SELECT name, collation_name FROM SYS.TYPES WHERE name = 'ct';
+GO
+~~START~~
+varchar#!#varchar
+ct#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+-- Case 4.1: Computed columns
+USE test_db_collation_vu_prepare_db121;
+GO
+
+SELECT * FROM test_db_collation_vu_prepare_db121_computed_columns ORDER BY a;
+GO
+~~START~~
+nvarchar#!#nvarchar
+Año Nuevo#!#oveuN oñA
+café#!#éfac
+chaptéR#!#Rétpahc
+película#!#alucílep
+~~END~~
+
+
+SELECT * FROM test_db_collation_vu_prepare_db121_computed_columns ORDER BY b;
+GO
+~~START~~
+nvarchar#!#nvarchar
+película#!#alucílep
+café#!#éfac
+Año Nuevo#!#oveuN oñA
+chaptéR#!#Rétpahc
+~~END~~
+
+
+-- Case 4.2: Temp Tables
+CREATE TABLE #test_db_collation_vu_prepare_db121_temp_t1(a nvarchar(11), b nvarchar(11), c AS sys.reverse(b) NOT NULL)
```

Comment:
Is it possible to check column metadata for this temp table?

Reply:
It's possible, but I think we do not populate the sys catalogs properly for temp tables. temp table do not come up in sys.tables. So did not add any tests for it. Will create an issue to track this.

Reply:
we can stil query pg_catalog for this purpose